dev/WebView2/WebView2.cpp

```diff
@@ -0,0 +1,1942 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include <corewindow.h>
+#include "dcomp.h"
+#include "InvokableCallbackHelper.h"
+#include "math.h"
+#include "ResourceAccessor.h"
+#include "RuntimeProfiler.h"
+#include "UIAutomationCore.h"
+#include "UIAutomationCoreApi.h"
+#include "WebView2.h"
+#include "WebView2AutomationPeer.h"
+#include "WebView2Utility.h"
+#include "Windows.Globalization.h"
+#include <wrl\event.h>
+
+// TODO: Lots of analyze warnings
+#pragma warning( disable : 26496 26462 26461 4471)
+
+#include <winuser.h>
+#include <uiautomationclient.h>
+
+using namespace Microsoft::WRL;
+
+WebView2::WebView2()
+{
+    if (HMODULE user32module = LoadLibraryW(L"user32.dll"))
+    {
```

Comment:
I know we're not targettng non-desktop right now but should this be using the apiset instead?
ext-ms-win-ntuser-window-l1-1-2 e.g.

Reply:
I'd rather not since we only care about desktop for now -- and we hope to be able to go back to static linking if we can get the WACK test relaxed.